uctMaths/competition/compadmin.py

```diff
@@ -630,6 +635,34 @@ def export_awards(request):
     output_workbook.save(response)
     return response
 
+def makeCertificates(students, assigned_school):
+    
+    certPath = "../Certificates/"
```

Comment:
Isn't there supposed to be another `..` here? There was an extra one before. Could you test that on your machine?

Reply:
I messed up the path the last time - I used two ..s last time as that is from the file itself, not from where the code is being run from, and the two pairs didn't work which made me replace it with the full path in the first place

Reply:
Hmm, initially I wasn't concerned about this because other parts of the code used relative directories. But now that there is confusion, I'm concerned.

Python has a special variable `__file__`, which contains the absolute path of the current file. So I think it would be safer if you did:

``` python
certPath = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(__file__))), 'Certificates')
```